.github/workflows/release.yml

```diff
@@ -150,9 +171,6 @@ jobs:
           artifacts: bitwarden_${{ env._PKG_VERSION }}_amd64.snap
           path: ./dist
```

Comment:
Shouldn't `artifacts` stay the same and the `path` key be changed to `./artifacts`?